### DISCUSSION TITLE
CSS styling

### DISCUSSION THREAD

**Discussion by cozats** [comfyanonymous/ComfyUI#3349] [Ideas]
Hey everyone, first time posting here. I don't much about coding, but I'm a designer so here's a little custom css for the menu, to make it a little bit better. I don't even know how to push it to main, so any suggestions are more than welcome.  Hope you like it! 
Add this to /web/user.css

```
 .comfy-menu {
    width: 220px;
}

.comfy-btn,
.comfy-menu>button,
.comfy-menu-btns button,
.comfy-menu .comfy-list button,
.comfy-modal button {
    font-size: .9em;
    padding-top: 8px;
    padding-bottom: 8px;
    width: 80%;
    border-radius: 5px;
    margin-bottom: 5px;
}


.comfy-queue-btn {
    font-weight: 700;
}
```

**Comment by lcolok** [comment]
I tried it out and found that this theme looks identical to the one used by the Twitter account I've been following and enjoying, [@Datou](https://twitter.com/Datou) ! I really like this theme!

**Comment by ThiagoFernandez** [comment]
for push it try to use the github desktop app

**Comment by ltdrdata** [comment]
Just fork this repo and clone it(your forked repo). And commit and push to your repo. Then you can see `Pull Request` button in your repo.

**Comment by joshgura** [comment]
tried this, and it worked. it's refreshing that my user.css works. Didn't feel like adding yet another thread on the subject (there's already several on reddit, etc) but  I've looked around for some way to increase the font size in the clip text encode area, tried to modify .json in the preferences, but it didn't have any effect that i could notice. 

also adding the following didn't have any effect:.
```
comfy-multiline-input, .litecontextmenu .litemenu-entry {
	font-size: 20px;
}`
```

**Reply by JohanK66** [reply]
To increase the font size of the prompt nodes I added this to the user.css

.comfy-multiline-input {
	font-size: 19px !important;
}

The !important instruction is ugly but often helpful when overriding CSS.

You might have to reset the server and refresh the browser to see the effect.

**Comment by BrechtCorbeel** [comment]
Are there any other CSS styling ideas on here? (I just did this its so beautiful thank you for doing this)

**Comment by hl24a3** [comment]
do you know the key name for a different font? the one they used is horrible to read

**Reply by cozats** [reply]
> do you know the key name for a different font? the one they used is horrible to read

Sorry I haven't seen your question. I really like what they did in the new version, but I'd suggest Inter form
Google fonts.

**Comment by christian-byrne** [comment]
For anyone arriving in future, please refer to docs on cusotmizing UI appearance and the new way to use the user.css file: https://docs.comfy.org/interface/appearance
